### ISSUE TITLE
Review of the Conversations feature

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#5784]
We released the [conversations](https://github.com/opencollective/opencollective/issues/2223) feature almost 3 years ago. It doesn't seem to be widely used, and @opencollective/design was discussing the eventuality of merging that feature with "Updates" at some point.

It would be great to check:
- the numbers: how many people are using this? Is the usage increasing?
- the use cases: who's using it, why?
- whether existing users are aware that this exists
- the user experience, to see if we're satisfied with the current version

With all that information, we should be able to decide on a long-term vision for this project.

**Comment by Memo-Es** [comment]
IMO conversations can be easily merged to updates by adding an ON/OFF switch to allow comments to each update. Having the same interface for both is weird, and merging them helps us align with our platform simplification goal.

**Comment by Memo-Es** [comment]
Related: https://github.com/opencollective/opencollective/issues/4179

**Comment by Betree** [comment]
@Memo-Es An important difference between conversations and updates is that anyone can start a conversation while an update is necessarily published by a collective admin. I agree the two features have many overlaps though; the only part that differs is the interface to post a new entry.

**Comment by Memo-Es** [comment]
@Betree, can we gather some data about how Collectives are using conversations? i.e., which Collectives have more conversations, and how many? Maybe comparing that with the same data on updates use.

**Comment by Betree** [comment]
> can we gather some data about how Collectives are using conversations? 

@Memo-Es Yep collecting some metrics about conversations usage is the purpose of this issue. I don't have much time to invest in it right now, just wanted to document the issue in case someone has more time to invest. Otherwise, I'll take a look later.

**Comment by Memo-Es** [comment]
That's fine @Betree, let me know if you need support on my side

**Comment by ntnsndr** [comment]
Thanks for the thoughtfulness here! I think some kind of conversational space is important, especially for enabling groups to make joint decisions about their resources. For instance, I am trying to use Open Collective to run both the funding and decision-making of a small office coffee-buying club:)

I can see a certain elegance with combining Conversations and Updates, though @Betree is right about what could be lost in terms of non-admin posting. What I'd suggest is something like this:

- Collapse the two into Conversations
- Enable Admins to check "send to all" for any posts if they want Update functionality
- Enable all users to create threads
- Enable users to opt-in to notifications on posts, or comments, or whatever
- Enable fine-grained granting of participation rights to certain "tiers" of contributors

Happy to discuss this more! I think this is a really important question, because it is about ensuring that resource decisions can be tied to community deliberation.
